tools/c7n_org/c7n_org/cli.py

```diff
@@ -360,6 +360,7 @@ def report_account(account, region, policies_config, output_path, cache_path, de
             r['policy'] = p.name
             r['region'] = p.options.region
             r['account'] = account['name']
+            r['account_id'] = account['account_id']
```

Comment:
this breaks report with other providers. all providers `accounts` have the `name` attribute but subscriptions and projects do not have account id and this would result in a key error when using report against those providers afaics.

Reply:
I think the below code has normalized the account conf of multi-cloud, after that, `account_id` will be available.

https://github.com/cloud-custodian/cloud-custodian/blob/c7c11b6c6580ca6a746d3e2cf2f1915580a62657/tools/c7n_org/c7n_org/cli.py#L565-L599

**Evidence:**
I added a below print at the first line of function `report_account` before running `c7n-org report -c azure-subs.yml ...`

```python
def report_account(account, region, policies_config, output_path, cache_path, debug):
    print(f"_report_account(): {account}")
    #  _report_account(): {'account_id': 'a1b2c3d4-e5f6-g7h8i9', 'name': 'Subscription-1', 'regions': ['global'], 'provider': 'azure', 'tags': (), 'vars': {}}_
```